instance/tests/utils.py

```diff
@@ -40,7 +39,11 @@ def patch_gandi(func=None):
     """
     Decorator to temporarily replace the Gandi API by the fake implementation.
     """
-    patcher = patch('instance.gandi.api', gandi.GandiAPI(client=FakeGandiClient()))
+    patcher = patch('instance.gandi.api', FakeGandiV5APIClient())
+    # mocked_set_dns_record = patch.object(gandi.GandiV5API, 'set_dns_record', MagicMock())
```

Comment:
Are these comments still needed? 
If the mocks are required, can you add a comments explaining, if not, can you clean them up?

Reply:
@giovannicimolin, I had added these while testing the changes. Will remove.